benchmarks/configs/infer_compositional_objects.py

```diff
@@ -60,34 +61,35 @@
 
 # For an explanation of the different levels of difficulty, see logos_on_objs.py
 
-model_path_monolithic_models_lvl1 = os.path.join(
-    pretrained_dir,
-    "supervised_pre_training_objects_with_logos_lvl1_monolithic_models/pretrained/",
+model_path_monolithic_models_lvl1 = (
+    pretrained_dir
+    / "supervised_pre_training_objects_with_logos_lvl1_monolithic_models/pretrained/"
```

Comment:
Could you take the `/`s out of the string and use the operator to join them? I know the original didn't do this, but the original was also problematic since `/` isn't guaranteed to be a path separator on all platforms.
```suggestion
    / "supervised_pre_training_objects_with_logos_lvl1_monolithic_models"
    / "pretrained"
```
Same with the others in this file.